src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTransformer.java

```diff
@@ -292,4 +434,27 @@ private void addApiImports(GapicInterfaceContext context) {
 
     return apiMethods;
   }
+
+  private String generateConfigOutputPath(GapicInterfaceContext context, String name) {
+    GapicInterfaceConfig interfaceConfig = context.getInterfaceConfig();
+    String outputPath =
+        pathMapper.getOutputPath(
+            context.getInterfaceModel().getFullName(), context.getProductConfig());
+    return outputPath.substring(0, outputPath.length() - 5)
```

Comment:
This looks odd to me - why do we need to manipulate the path like this?

Reply:
Originally, the code looked like this:

```java
return outputPath
        + "/../resources/"
        + Name.upperCamel(interfaceConfig.getInterfaceModel().getSimpleName())
            .join(name)
            .toLowerUnderscore()
        + ".php";
```

However, at generation time I was receiving the following error:

`java.io.FileNotFoundException: /Users/dave/dev/google-php-cloud-speech-v1beta1/src/Speech/V1beta1/Gapic/../resources/speech_rest_client_config.php (No such file or directory)`

This workaround solved the issue for me, however, I am looking into some other cleaner options :).